Realm/Realm/Linq/WhereClauseVisitor.cs

```diff
@@ -0,0 +1,329 @@
+            //                {
+            //                    var leftName = GetColumnName(me, me.NodeType);
+
+            //                    //stringComparisonNode.PropertyNode.Property = leftName;
+            //                    //stringComparisonNode.PropertyNode.Type = "string";
+            //                }
+            //            }
+
+            //            return stringComparisonNode;
+            //        }
+            //        else if (AreMethodsSame(expMethod.Method, Methods.String.StartsWithStringComparison.Value))
+            //        {
+            //            Console.WriteLine("Test");
+            //        }
+            //        else if (AreMethodsSame(expMethod.Method, Methods.String.EndsWith.Value))
+            //        {
+            //            Console.WriteLine("Test");
+            //        }
+            //        else if (AreMethodsSame(expMethod.Method, Methods.String.EndsWithStringComparison.Value))
+            //        {
+            //            Console.WriteLine("Test");
+            //        }
+            //        else if (AreMethodsSame(expMethod.Method, Methods.String.IsNullOrEmpty.Value))
+            //        {
+            //            Console.WriteLine("Test");
+            //        }
+            //        else if (AreMethodsSame(expMethod.Method, Methods.String.EqualsMethod.Value))
+            //        {
+            //            Console.WriteLine("Test");
+            //        }
+            //        else if (AreMethodsSame(expMethod.Method, Methods.String.EqualsStringComparison.Value))
+            //        {
+            //            Console.WriteLine("Test");
+            //        }
+            //        else if (AreMethodsSame(expMethod.Method, Methods.String.Like.Value))
+            //        {
+            //            Console.WriteLine("Test");
+            //        }
+            //    }
+            //}
+
+            return RealmLinqExpression.Create(returnNode);
+        }
+
+        private string GetColumnName(MemberExpression memberExpression, ExpressionType? parentType = null)
+        {
+            var name = memberExpression?.Member.GetMappedOrOriginalName();
+
+            if (parentType.HasValue)
+            {
+                if (name == null ||
+                    memberExpression.Expression.NodeType != ExpressionType.Parameter ||
+                    !(memberExpression.Member is PropertyInfo) ||
+                    !_metadata.Schema.TryFindProperty(name, out var property) ||
+                    property.Type.HasFlag(PropertyType.Array) ||
+                    property.Type.HasFlag(PropertyType.Set))
+                {
+                    throw new NotSupportedException($"The left-hand side of the {parentType} operator must be a direct access to a persisted property in Realm.\nUnable to process '{memberExpression}'.");
```

Comment:
For the same reason I've said before, I think that this is not necessarily the left-hand side